src/lightbox/shareFile.js

```diff
@@ -0,0 +1,41 @@
+/* @flow strict-local */
```

Comment:
I think this file should still be `shareImage` rather than `shareFile`, since it's lightbox-specific — the lightbox only shows images, not other types of files.

That'll also make the diff a bit nicer for me to look at :)

Reply:
Right makes sense, refactored accordingly.